consensus/polybft/consensus_runtime_test.go

```diff
@@ -389,11 +397,10 @@ func TestConsensusRuntime_FSM_EndOfEpoch_BuildCommitEpoch(t *testing.T) {
 	validatorAccounts := validator.NewTestValidatorsWithAliases(t, []string{"A", "B", "C", "D", "E", "F"})
 	validators := validatorAccounts.GetPublicIdentities()
 
-	lastBuiltBlock, headerMap := createTestBlocks(t, 9, epochSize, validators)
+	lastBuiltBlock := &types.Header{Number: 9}
```

Comment:
Nitpick, we can spare this line, by inlining it when instantiating `consensusRuntime` :wink:

Reply:
Done: https://github.com/0xPolygon/polygon-edge/pull/1735/commits/7fa9afd2985856e7b552d15dc83aa367daa78411